src/index.js

```diff
@@ -30,24 +26,26 @@ class Experiment extends React.Component {
   }
 
   fetchVariantName() {
-    return this.props.fetchVariantName && this.props.fetchVariantName(this.experimentId)
+    return this.props.fetchVariantName && this.props.fetchVariantName(this.props.experimentId)
```

Comment:
what does `this.props.fetchVariantName` return? I didn't find where the component render `this.props.children` 🤔

Reply:
Ah, never mind. It's from the `Variant` component.

Reply:
@fraserxu , I guess this `this.props.fetchVariantName` is a check that there is a function `fetchVariantName` passed to `Experiment` as a property

Reply:
yep, sorry. I misread the `findVariantByName` as `fetchVariantName `.